composer.json

```diff
@@ -10,20 +36,20 @@
   "require": {
     "dompdf/dompdf" : "0.7.*",
     "electrolinux/phpquery": "^0.9.6",
-    "symfony/config": "~2.5.0",
-    "symfony/dependency-injection": "~2.5.0",
-    "symfony/event-dispatcher": "~2.5.0",
-    "symfony/filesystem": "~2.5.0",
-    "symfony/process": "~2.5.0",
-    "psr/log": "1.0.0",
-    "symfony/finder": "~2.5.0",
+    "symfony/config": "^2.5.0",
```

Comment:
We can keep `~` but use `||` to allow different? so `~2.5 || ~2.8` to compatibility with Drupal. But also, as stated in StackExchange.. 2.5 is outdated and not receiving releases.